### ISSUE TITLE
Mac os patch install error

### ISSUE THREAD

**Issue by AyYildizPilot** [dosdude1/macos-catalina-patcher#222]
THIS IS NOT YOUR HELPDESK, WE HAVE FORUMS FOR A REASON. Please only file an issue here for code-related problems. This is not an issue system for resolving issues with using the compiled binaries, or discussing functionality issues with Catalina on your unsupported machine.
- [ ] I understand

Hello,
Mac OS X version: macOS Catalina Patcher in 10.7.2 won't open after double click. It seems to appear in the status bar and disappears.
https://dosdude1.com/highsierra/
High Sierra SUB disc creation was successful.
I installed the system. After that it won't open.
I tried both Mac OS Extended (Journaled) and APFS boot, patched with macbookair 2.1, unfortunately it won't boot.

What do I need to do to get this device up?

MacbookAir late 2008 A1304
MacbookAir2.1
Intel Core 2Duo 1.07GHz
2GB RAM
System: Mac OS X version: 10.7.2

**Comment by AyYildizPilot** [comment]
Hello again,

No Catalina with Mac OS X version: 10.7.2. I created a High Sierra patch USB Disk, I did the patch before the installation.
I did the installation The pc didn't turn on, then I patched it again and it's working now.

Thanks Dosdude1.

My Hardware Is Low Which Mac OS version is running fast? Do you have any advice?
macOS Catalina Patcher
macOS Mojave Patcher
macOS High Sierra Patcher
macOS Sierra Patcher

**Comment by AyYildizPilot** [comment]
My problem is solved.
I reviewed the steps at https://dosdude1.com again.

I installed macOS High Sierra Patcher and worked smoothly on this old computer.

I installed macOS Mojave Patcher and it worked.

I installed macOS Catalina Patcher and it worked.

Macbookair 2.1 late 2008 2gb ram very old and slow Computer so catalina installed too late. But the problem in its work is that it works very well and stable.

Thanks Dosdude The old computer came to life.
